### PR TITLE
Cb admin dashboard

### PR DESCRIPTION
**Pull request by eliascodes**
CB admin dashboard styled + dots added to all pages.

Buttons on the dashboard are slightly off center, but at this stage, CBA, would rather finish the other views.

I decided against changing the auth system; I started, but it turned into this huge thing, so better leave it till later.

### CODE REVIEW

File: react-backend/client/src/shared/components/Dots.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+
+export default styled.div`
+  z-index: -1;
+  left: 50%;
+  top: 0;
+  transform: translate3d(-50%, 0, 0);
+
+  &, &::before, &::after {
```
Comment (RogeredBacon) [inline react-backend/client/src/shared/components/Dots.js]: What does this do?

Comment (eliascodes) [reply react-backend/client/src/shared/components/Dots.js]: I don't know, I just copied it from their website 🤷‍♂️

Comment (RogeredBacon) [reply react-backend/client/src/shared/components/Dots.js]: Haha coding in a nutshell

File: react-backend/client/src/App.js [hunk 2]
```diff
@@ -56,154 +57,156 @@ class App extends Component {
...
-            path="/admin/feedback"
-            updateLoggedIn={this.updateLoggedIn}
-            component={adminLoginCheck(CbAdminFeedbackPage)}
-          />
-
-          <Route exact path="/internalServerError" component={InternalServerError} />
-          <Route component={NotFound} />
-        </Switch>
-      </Container>
+      <Dots>
```
Comment (astroash) [inline react-backend/client/src/App.js]: How come there's such a big dif here? Can this be avoided as this caused me some major rebase woes last time.

Comment (eliascodes) [reply react-backend/client/src/App.js]: An extra wrapping element means the indentation level for all children increases by one.

Comment (eliascodes) [reply react-backend/client/src/App.js]: What changes did you make to the App.js? Just swap out some components in the route definitions? If so, handling conflicts on rebase is simple: accept the current changes (i.e. changes _not_ coming from your commits), then past your components back in.

**Pull request merged**